makefile.common

```diff
@@ -0,0 +1,85 @@
+#common make targets shared over multiple makefiles
```

Comment:
I would be explicit `#common make targets shared by makefile + makefile.shared (GNU make only)`